src/harness/harness.ts

```diff
@@ -18,12 +18,13 @@
 /// <reference path="..\services\shims.ts" />
 /// <reference path="..\server\session.ts" />
 /// <reference path="..\server\client.ts" />
-/// <reference path="..\server\node.d.ts" />
-/// <reference path="external\mocha.d.ts"/>
-/// <reference path="external\chai.d.ts"/>
 /// <reference path="sourceMapRecorder.ts"/>
 /// <reference path="runnerbase.ts"/>
 /// <reference path="virtualFileSystem.ts" />
+/// <reference types="node" />

```

Comment:
Strictly speaking you should not, since it's in the config file. Not for the compilation, anyway. However for the editor experience, if the editor starts from the unittests folder and walks back to the harness (via the /// ref in the unittest), it doesn't always see the tsconfig in the harness folder and associate the unittest with that project (so adding the references into the harness file which gets included in all unittests covers it, making the editor experience without it finding the right tsconfig acceptable).

Only did this because it happened to me when reopening the editor while working on this.